src/category_theory/monad/coequalizer.lean

```diff
@@ -36,26 +35,27 @@ Show that any algebra is a coequalizer of free algebras.
 
 /-- The top map in the coequalizer diagram we will construct. -/
 @[simps {rhs_md := semireducible}]
-def coequalizer.top_map : (monad.free T).obj (T.obj X.A) ⟶ (monad.free T).obj X.A :=
+def free_coequalizer.top_map : (monad.free T).obj (T.obj X.A) ⟶ (monad.free T).obj X.A :=
```

Comment:
Please mention that you rename these declarations in the top post of your PR. That way if people search the git log they can find what's going on.

Reply:
Done.